R/request.R

```diff
@@ -122,6 +122,12 @@ request_prepare <- function(req) {
 
 request_perform <- function(req, handle, refresh = TRUE) {
   stopifnot(is.request(req), inherits(handle, "curl_handle"))
+
+  orig_req <- req
```

Comment:
Sure. Btw. I was also thinking that probably we should use the request as it is sent out, i.e. after `request_prepare`, no?